python/packages/ai/teams/ai/models/openai_model.py

```diff
@@ -232,10 +251,17 @@ async def complete_prompt(
                     content=msg.content if msg.content else "",
                 )
             elif msg.role == "system":
-                param = chat.ChatCompletionSystemMessageParam(
-                    role="system",
-                    content=msg.content if msg.content is not None else "",
-                )
+                # o1 models do not support system messages
+                if is_o1_model:
```

Comment:
please add a test case in the unit tests for this

Reply:
added